homeassistant/components/tesla/translations/de.json

```
@@ -13,7 +13,8 @@
             "user": {
                 "data": {
                     "password": "Passwort",
-                    "username": "E-Mail"
+                    "username": "E-Mail",
+                    "mfa": "Multi-Faktor Code (Optional)"
```

```suggestion
                    "username": "E-Mail"
```

This change needs to happen on lokalise as it will get overwritten here.

https://developers.home-assistant.io/docs/translations/

---

Ah, thank you. Did not know that